pkg/controller/pod.go

```diff
@@ -56,12 +56,21 @@ func (n *NamedPort) AddNamedPortByPod(pod *v1.Pod) {
 	ns := pod.Namespace
 	podName := pod.Name
 
-	if pod.Spec.Containers == nil {
+	sidecarContainers := make([]v1.Container, 0, len(pod.Spec.InitContainers))
```

Comment:
```suggestion
	initContainers := make([]v1.Container, 0, len(pod.Spec.InitContainers))
```
sidecar 和 initContainer 不是一个概念

Reply:
Fixed.